pkg/dispatchcli/cmd/create.go

```diff
@@ -63,6 +66,39 @@ func importFile(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []stri
 	return importBytes(out, b, actionMap, actionName)
 }
 
+func importFileWithURL(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []string, actionMap map[string]ModelAction, actionName string) error {
+	err := downloadFile(file, tmpPath)
```

Comment:
Actually, you don't even have to download the file to tmp dir, you could do this in-memory by reading the response and passing to `importBytes`